app/models/resource.py

```diff
@@ -5,11 +5,25 @@ class Resource(db.Model):
     __tablename__ = 'resources'
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(64))
-    address = db.relationship('Address', backref='resource', lazy='joined',
-                              uselist=False)
     description = db.Column(db.Text)
     website = db.Column(db.Text)
+    address_id = db.Column(db.Integer, db.ForeignKey('address.id'))
     reviews = db.relationship('Review', backref='resource', lazy='dynamic')
 
     def __repr__(self):
         return '<Resource \'%s\'>' % self.name
+
+
+class ResourceReview(db.Model):
+    __tablename__ = 'resource_reviews'
+    id = db.Column(db.Integer, primary_key=True)
+    timestamp = db.Column(db.DateTime)
+    content = db.Column(db.Text)
+    rating = db.Column(db.Integer)
```

Comment:
What are the constraints of the rating? Maybe throw a comment on the field describing it a little more.